invadedhouse/commands.py

```diff
@@ -24,7 +28,10 @@ def get_next_command() -> str:
         return cmd
 
     def interpret_command(self, cmd: str):
-        if cmd == 'exit':
+        if cmd in ('exit', 'bye', 'i\'m leaving', 'i\'m gonna exit', 'end game', 'quit', 'goodbye', 'leave', 'leave so soon', 'delete my progress', 'get outta here', 'skedaddle',
+        'let me out', 'stop', 'end', 'avengers endgame', 'im scared of this place i wanna go home', 'i wanna go home', 'i want my mommy', 'i want my daddy', 'im out', 'goodbye',
+        'this game is trash', 'i dont like this game', 'i dont like this', 'stop running run.py', f'go from {self.player.get_location()} to my house', 'walk into the strange portal that is on the wall that leads to my home',
+        'when is this going to end', f'create a portal to my house in {self.player.get_location()}', 'i hate you', 'exit the game'):
```

Comment:
all of the things you can type in instead of "exit"

Reply:
my favorite one is `f'create a portal to my house in {self.player.get_location()}'`.